client/my-sites/hosting/main.js

```diff
@@ -238,6 +242,7 @@ export default connect(
 	( state ) => {
 		const siteId = getSelectedSiteId( state );
 		const hasSftpFeature = siteHasFeature( state, siteId, FEATURE_SFTP );
+		const hasStagingSites = siteHasFeature( state, siteId, FEATURE_SITE_STAGING_SITES );
```

Comment:
Can we make it more explicit that the site has the staging sites "feature" ?

```suggestion
		const hasStagingSitesFeature = siteHasFeature( state, siteId, FEATURE_SITE_STAGING_SITES );
```